README.md

```diff
@@ -1 +1,57 @@
-# MobileVersionSkeletonProject
\ No newline at end of file
+# MobileVersionSkeletonProject
+
+This is the second project at Microverse first week activity and my portfolio website.
+
+## Built With
+
+- HTML 
+- CSS
+
+## Getting Started
+
+in the process of creating this portfolio website,set up a new repository and prepare it for development using best practices like linters. Build the first 2 sections of the mobile website using the template in figma.
+
+## Prerequisites
+
+- Install editor like Vscode
+- Install broweser for seeing the result
+- Understand the concept of version control
+- GitHub account
+- Know GitHub flow
+- Set up GitHub action
+- Set up linters
+- Understand basic HTML
+- Understand Basic CSS
```

Comment:
- [ ] You have done well modifying your README according to the requested changes. In the **Getting Started** section, you should tell the user how to clone your repo. Something like this 👇 
```
git clone https://github.com/aster-alemu/MobileVersionSkeleton.git
```

- [ ] In the **Prerequisites** section, just list the necessary things needed for a cloned copy of your project to run smoothly. All those things you listed are not necessary for your project to run correctly.
**Note** if there are no prerequisites, just leave that part blank 👍